### DISCUSSION TITLE
Langton's Ant Cellular Automata for Minimal64x4

### DISCUSSION THREAD

**Discussion by bobbarnes1981** [slu4coder/Minimal-64x4-Home-Computer#14] [Show and tell]
Here is my implementation of [Langton's Ant](https://en.wikipedia.org/wiki/Langton%27s_ant) for the Minimal 64x4

https://github.com/bobbarnes1981/M64x4LangtonsAnt/tree/main

I think there are probably some more efficiencies that can be gained, for example; it will check every single cell in the grid but it only really needs to check the current cell where the ant lives.

**Comment by slu4coder** [comment]
Hi Bob, this looks great. I just uploaded the code and generated some awesome patterns :-)
